### ISSUE TITLE
USB audio still broken

### ISSUE THREAD

**Issue by giddyhup** [raspberrypi/linux#586]
The recent update from May 1 initially looked like an improvement. Even my USB connected DPF became stable. Yet, with my USB sound dongle some sounds sounded odd or wouldn't play at all. dmesg showed some NYET messages related to the sound card, I had to disable the driver again.

```
[  141.005422] Transfer to device 7 endpoint 0x1 frame 1150 failed - FIQ reported NYET. Data may have been lost.
[  141.325427] Transfer to device 7 endpoint 0x1 frame 1470 failed - FIQ reported NYET. Data may have been lost.
```

NYET errors keep happening during audio playback. Please continue here or re-open #575. Thanks.

**Comment by P33M** [comment]
Please post a full `lsusb -v` and `dmesg` since boot.

Edit: also, please post a `lsusb -t` which shows your hierarchical connection of devices.

**Comment by giddyhup** [comment]
https://gist.github.com/giddyhup/a6971b081efea564af00

Probably nothing, just for the record, I got some errors creating the lsusb -t output:

```
$ sudo lsusb -t > lsusb-t.txt
1-1.3.1.1:1.0: No such file or directory
1-1.3.1.1:1.1: No such file or directory
1-1.3.1.3:1.0: No such file or directory
```

**Comment by P33M** [comment]
Is your USB audio device still broken if you remove all other USB devices (save keyboard) and do e.g. `speaker-test -t sine -f 1000 [...]`?

**Comment by giddyhup** [comment]
I will test that. It needs some preparation since my Raspberry is headless.

**Comment by giddyhup** [comment]
Tested it, got no NYET errors with just a keyboard and the soundcard attached. Since this was not an actual use case I stepped it up and connected the wifi dongle in place of the keyboard. Still no NYET errors. I then rotated many things around and the current state of research seems to be that the Blutooth dongle might be the culprit/cause for the NYET errors. The following are the entries before and when I removed the dongle:

```
[  346.634373] Transfer to device 7 endpoint 0x1 frame 2030 failed - FIQ reported NYET. Data may have been lost.
[  364.465714] usb 1-1.3.3: USB disconnect, device number 7
```
